c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl

```diff
@@ -491,7 +491,11 @@ endif
 build: $(BUILD_RULES)
 
 .PHONY: checks
-checks: flake8 $(CLIENT_CHECK_RULE) $(WEB_RULE)
+checks: flake8 $(CLIENT_CHECK_RULE) $(PACKAGE)/static-ngeo/build/main.js $(PACKAGE)/static-ngeo/build/main.css git-attributes
```

Comment:
To be more generic:
`$(PACKAGE)/static-ngeo/build/main.js $(PACKAGE)/static-ngeo/build/main.css`
=>
```
$(addprefix $(APP_OUTPUT_DIR)/, $(addsuffix .js, $(NGEO_INTERFACES))) \
$(addprefix $(APP_OUTPUT_DIR)/, $(addsuffix .css, $(NGEO_INTERFACES))) \
```
And to avoid copy past we should create one or two variable.
https://github.com/camptocamp/c2cgeoportal/blob/b058d86a4759ab3021c505e93a6808f43e6c533e/c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl#L193-L194